docs/tutorials/deploy-contracts.mdx

```diff
@@ -113,4 +113,20 @@ INFO [03-26|09:59:54.515] Contract deployed                        address=0xFcd
 
 Congratulations! You just deployed your first contract to a local SUAVE network! 💃 
 
-Take note of the address to which it has been deployed: you'll need it in the next tutorial, where we'll send confidential compute requests to your contracts, and extend the functions to emit information about offchain computations onchain.
\ No newline at end of file
+Take note of the address to which it has been deployed: you'll need it in the next tutorial, where we'll send confidential compute requests to your contracts, and extend the functions to emit information about offchain computations onchain.
+
+### Pre-funded accounts
```

Comment:
this section almost feels like it deserves its own page. Or turn it into a component and include it in multiple pages.
I find my self fetching these keys/addresses all the time and need them easy-to-find